lib/client.js

```diff
@@ -45,8 +45,17 @@ function timeoutReject (reject, name, client, paramsArray) {
   }
 }
 
+export function stripActionArgs(action) {
+  // Capture:
+  // 1. single or multiple comma-delimited arguments following an initial colon, and
+  // 2. non-whitelisted words following a period (i.e. file extensions in the case of assetURL calls)
+  const ACTION_ARGS = /\:\w+(,?\w+)*((\.(show|hide|enable|disable))|(\.\w*))?/g;
+  // Return :arg.field_modifier, where field modifier is one of 'show', 'hide', 'enable' or 'disable'
+  return action.replace(ACTION_ARGS, ':arg$3')
+}
+
 function defaultTimer (client, actions, callback) {
-  const actionsTags = actions.map(action => `action: ${action}`)
+  const actionsTags = actions.map(action => `action:${stripActionArgs(action)}`)
```

Comment:
⛏ space between `action:` and  `${stripActionArgs(action)}` to be consistent with existing (few days worth of) events?

Reply:
Intentional change. Datadog isn't very useful for looking back far at historical data, and the space just sticks an unwanted underscore character in there. I think we'd be fine to ignore the days worth of data on 20/02 and work from the point that this gets merged onwards